### ISSUE TITLE
Increase line height of accordion text for legibility

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#20]
# Increase line height of accordion text for legibility

## Defect level
Feedback will follow a must, should, consider feedback framework:

- ❗️ **Must for if the feedback must be applied.** Failure to follow this feedback may either cause harm to users, block a disabled person from completing an important task, or result in legal risk (fails to meet WCAG 2.1 AA compliance).
- ⚠️ **Should if the feedback is best practice.** Following this feedback will significantly improve the usability for disabled people or reduce the complexity of accessibility barriers.
- ✔️ **Consider for suggestions or enhancements.** Think of these suggestions as reducing the hundreds of papercuts and microaggressive barriers disabled people may face when navigating the web. Going above and beyond here will make us a leader in accessibility. 

## Description
The current line height for accordion text is a smidge small. Increasing the line height will improve legibility! Definitely a consider tho, so only take this up if you have extra time.

## Before
<img width="373" alt="Screen Shot 2022-05-24 at 6 31 47 PM" src="https://user-images.githubusercontent.com/14154792/170142644-b3e78049-378f-4a99-9b4a-a5f2d9f8654f.png">

## After (around 26px)
<img width="373" alt="Screen Shot 2022-05-24 at 6 31 36 PM" src="https://user-images.githubusercontent.com/14154792/170142630-020c0a49-d619-4837-add4-cdd830ba8d58.png">
